<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QDialog" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>настройки</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>шрифт </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>размер шрифта</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="fontSize"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>тема</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="colorTheme">
     <item>
      <property name="text">
       <string>Dark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Classic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>подсветка синтасиса</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="syntax">
     <item>
      <property name="text">
       <string>обычный текст</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C++</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>python</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>перенос строки</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="wrapMode">
     <item>
      <property name="text">
       <string>вкл</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выкл</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
